### DISCUSSION TITLE
My new project (now_in_dart_flutter)

### DISCUSSION THREAD

**Discussion by Biplab-Dutta** [isar/isar#712] [Show and tell]
Hey all. I have just open-sourced my new flutter pet project (now_in_dart_flutter). As the name suggests, the app intends to make it easy for all the users to easily be able to see about latest changes in the Dart and Flutter ecosystem. The uses Isar for offline support.

Here's a demo.

https://user-images.githubusercontent.com/63902683/191918822-a06bce7d-16a5-4a2d-af38-4d20465472fa.mp4

🔗 [Source code](https://github.com/Biplab-Dutta/now-in-dart-flutter)

🔗 [Download APK from GitHub Release](https://github.com/Biplab-Dutta/now-in-dart-flutter/releases/tag/v1.0.0)

**Comment by muddassar-lab** [comment]
I am happy to see that more and more people are liking and adopting isar for their old and new flutter projects.
